package.json

```diff
@@ -15,15 +15,22 @@
     "@typescript-eslint/eslint-plugin": "^5.40.1",
     "@typescript-eslint/parser": "^5.40.1",
     "axios": "^0.21.4",
+    "buffer": "^6.0.3",
+    "crypto-browserify": "^3.12.0",
     "eslint-config-airbnb": "^19.0.4",
     "eslint-config-airbnb-typescript": "^17.0.0",
     "history": "^4.10.1",
+    "jose": "^4.11.0",
+    "jsonwebtoken": "^8.5.1",
```

Comment:
I think we don't need `jsonwebtoken` (and related dependencies) now that we're using `jose` right?